### DISCUSSION TITLE
Physics-based preconditioner (PBP)  parameter settings

### DISCUSSION THREAD

**Discussion by Richard-happy** [idaholab/moose#23779] [Q&A General] [answered]
Which options does the parameter about **preconditioner** have?
![image](https://user-images.githubusercontent.com/40360358/226186811-f5d64845-bb53-4b3a-bcff-4e892ae1ffb1.png)

**Comment by GiudGiud** [comment, marked as answer]
Hello

it will take other pre conditioners defined in your Preconditioning block. 

The idea is to create a different preconditioner each variable or groups of variables

Guillaume

**Reply by Richard-happy** [reply]
So far I know it has options such as **AMG LU ILU ASM**.Are there some other options I could choose?Or any codes I could refer to?

**Reply by GiudGiud** [reply]
https://mooseframework.inl.gov/source/executioners/Steady.html

There s a list of options there for preconditioning 

However this page does not use the Preconditioning syntax so you ll have to look at that page to see those

**Reply by Richard-happy** [reply]
Thank you
